docs/our_oss_project.md

```diff
@@ -9,96 +9,11 @@ hide:
 
 ### **BeSman**
 
-<div align="justify">
-One of the first utilities to be created as part of the Be-Secure project is BeSman.
-<p><p>
-BeSman is a command-line utility to provision customized environments for each open source tech stack. These environments are known as BeSman environments. We have two types of BeSman environments – dev and sec environments. 
-<p>
-The development environment is pre-bundled with all tools and dependencies that a developer would need to work on a specific open source project. Similarly, the sec environments are pre-bundled with a set of open source security tools that a security tester can leverage to conduct planned security assessment. 
-</div>
- 
-**Why do we need BeSman utility**
-
-<div align="justify">
-Individuals spend considerable effort to set up and configure the open source project in their local environment to evaluate it / commence working on it. At times individuals run into configuration issues or set up issues which result in them spending more effort to resolve them. This is a sizeable effort that is spent just to get the environment up as compared to the effort spent to build newer capability using an existing open source project. This is a misspend effort that should be managed in a better way. 
-The BeSman utility provides command-line capability to provision customized environments quickly and in a consistent manner. 
-</div>
- 
-**How can BeSman utility help**
-
-<div align="justify">
-BeSman environments are pre-bundled with tools and dependencies for a specific open project and it can be provisioned using simple bash shell commands from BeSman utility.
-
- 
-BeSman utility will also provide configurable capabilities that would permit the user to configure the tools and dependencies to be pre-bundled in the environment.
-
- 
-Individuals will have complete control over what goes into building and provisioning the environment.
+BeSman is a command-line utility to provision customized environments for each open source tech stack. 
```

Comment:
I believe this line is getting repeated in the BeSman's file.

Reply:
@asa1997  
Updated The project description. Please check and share your comments. 

 @panickervinod  @sumodgeorge